components/core/Helpers/JsonElementHelper.cs

```diff
@@ -3,15 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
-using System.Text;
 using System.Text.Json;
 
 namespace AntDesign.Core.Helpers
 {
-    public static class JsonElementHelper<TValue>
+    internal static class JsonElementHelper<TValue>
```

Comment:
There are a couple classes that were changed to internal or private that I'd recommend keeping public. If someone is using them in their project it will break. I think making them internal is fine but in a breaking change release

Reply:
Thanks for your review!  `JsonElementHelper<TValue>` and `THelper` (what a name 🤣 ) are absolutely not suitable for public exposure as they're highly encapsulated helpers which aim to only serve for this lib and external users should not rely on them.

> I think making them internal is fine but in a breaking change release

You're right. The release note should notice this if it's approved. And for users who already use them, the workaround is easy: make themself sure they do want to use the codes , and copy them then.

Anyway, I need to discuss with @ElderJames . If he is more inclined to your opinion, I'll roll the metioned changes back.

Reply:
This pr is better to focus on the test project first and not add changes.

Reply:
Well, I'll handle it tonight.